clientv3/op.go

```diff
@@ -415,6 +424,22 @@ func WithCreatedNotify() OpOption {
 	}
 }
 
+// WithFragmentedResponse makes the watch server send watch responses
+// that are too large to send over the rpc stream in fragments.
+// The fragmenting feature is an opt-in feature in order to maintain
+// compatibility with older versions of clients.
+func WithFragmentedResponse() OpOption {
```

Comment:
Can remove this since the client here can default to fragmenting. If it's an old server, it will ignore the flag and default to non-fragmenting. For backwards compatibility, it only needs to be optional on the server side.

Reply:
I'm a little confused, what's the point of the fragment flag if I remove the code to toggle the flag? Won't it always be true?